sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -2247,6 +2277,7 @@ private void SetNameFieldsIfNull()
                 progressHandler,
                 $"{nameof(BlobBaseClient)}.{nameof(DownloadContent)}",
                 async: async,
+                false,
```

Comment:
I think we want `DownloadContentInternal` to take upn as a parameter.

Reply:
DownloadContentInternal is used by DownloadContent API in the blobs package. This API is not utilized by datalake package for file downloads. Hence not adding upn here and passing/hardcoding false under the hood to internal methods that are common amongst DownloadStreamingInternal (the internal method called by the downloaders that files use from the blobs package) and DownloadContentInternal.